installer/installer.py

```diff
@@ -267,7 +292,10 @@ def get_binary(self):
         try:
             self.exec(f"wget -c {binary_url}")
             if fname.find(".tar.gz") != -1:
-                self.exec(f"tar -xzf {fname} -C .")
+                if self.version.replace('v', '') == '0.6.9':
```

Comment:
Special case for v0.6.9 is probably not needed any more because latest in that release family is v0.6.10 and URL structure is same as all other version

Reply:
Done! ✅